tests/test_umls_utils.py

```diff
@@ -0,0 +1,36 @@
+# pylint: disable=no-self-use,invalid-name
+import unittest
+
+from scispacy import umls_utils
+
+class TestUtil(unittest.TestCase):
+
+    def test_read_umls_concepts(self):
```

Comment:
for these tests, I would probably make the fixture smaller and then assert that you get exactly what you expect